tools/build_release.py

```diff
@@ -9,7 +9,8 @@
 pp = pprint.PrettyPrinter(indent=4, width=100)
 
 default_python_cmd = ['python.exe']
-drivers_to_release = ['nifake', 'nidcpower', 'nidigital', 'nidmm', 'niswitch', 'nimodinst', 'nifgen', 'niscope', 'nise', 'nitclk']
+drivers_to_update = ['nifake', 'nidcpower', 'nidigital', 'nidmm', 'niswitch', 'nimodinst', 'nifgen', 'niscope', 'nise', 'nitclk']
+drivers_to_upload = ['nidcpower', 'nidigital', 'nidmm', 'niswitch', 'nimodinst', 'nifgen', 'niscope', 'nise', 'nitclk']
```

Comment:
If the lists are identical other than for `nifake` I think you should base one variable off the other.

Reply:
Done